bull-common/src/test/java/com/hotels/transformer/utils/ClassUtilsTest.java

```diff
@@ -997,12 +1003,97 @@ public void testGetConcreteClassWorksAsExpected(final String testCaseDescription
         };
     }
 
+    /**
+     * Test that the a {@link MissingMethodException} is raised if the Builder class has no {@code build()} method.
+     * @param testCaseDescription the test case description
+     * @param containerTestClass the test class containing the Builder
+     * @param builderClass the Builder nested class
+     */
+    @Test(dataProvider = "dataGetBuilderMethodExceptionTesting", expectedExceptions = MissingMethodException.class)
+    public void testGetBuildMethodThrowsExceptionIfMethodIsMissing(final String testCaseDescription, final Class<?> containerTestClass, final Class<?> builderClass) {
```

Comment:
The parameter`testCaseDescription` is not being used anywhere

Reply:
yes it is, is the first argument passed by the data provider. That description is shown at compile time in the console and in your IDE while running the test. It helps the troubleshooting